packages/extensions/venia-sample-backends/__tests__/__snapshots__/intercept.spec.js.snap

```diff
@@ -13,9 +13,3 @@ exports[`should call onFail with a different error message if environments is em
  [{\\"name\\":\\"2.3.3-venia-cloud\\",\\"description\\":\\"Magento 2.3.3 with Venia sample data installed\\",\\"url\\":\\"https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/\\"}] 
```

Comment:
This seems incorrect. If the environments is empty, where is this "other backend" data coming from? I'll have to fix the test.

Reply:
Fixed. fetch wasn't updating mock for some reason. Had to move test order.